odoo/addons/base/models/res_config.py

```diff
@@ -591,7 +591,7 @@ def set_values(self):
         for name, icp in classified['config']:
             field = self._fields[name]
             value = self[name]
-            current_value = current_settings[name]
+            current_value = IrConfigParameter.get_param(icp, default=False)
```

Comment:
```suggestion
            IrConfigParameter.get_param(icp)
```
`False` is already the default value of `default` argument.